### ISSUE TITLE
Random rendering issues on multiple Mac devices

### ISSUE THREAD

**Issue by Kimmichka** [pubpub/pubpub#247]
Hello,

I'm part of a collaborative publishing teamwork here on PubPub and have encountered random rendering issues on multiple Mac devices on 4 very specific pubs of our journal (cf. screenshots below):
https://www.pubpub.org/pub/01-psychoanalysis-chapter
https://www.pubpub.org/pub/03-institutional-therapy
https://www.pubpub.org/pub/04-client-centred-psychotherapy
https://www.pubpub.org/pub/05-cognitive-remediation-therapy

While I have no problem viewing these 4 pubs on the following 2 Mac devices:
- iPhone 6S Plus - iOS 10.3.1
- Mac Mini mid-2011 - OS 10.11.6

the pubs would not render at all on the following 7 devices:
- iPad 3 - iOS 7.1.2
- iPad 3 Mini - iOS 9.3.5  
- iPhone 5S - iOS 10.1.1
- iPad Mini - iOS 10.2
- iPhone SE - iOS 10.3.2
- MacBook Air 2012 - OS 10.10.5
- MacBook Air mid-2011 - OS 10.11.4

I have tried to access these 4 pubs with 3 different browsers: Safari, Firefox and Chrome. I have emptied all the browsers cache, as well as the DNS cache. I have also emptied my browser cache on the iPhone SE and tried to connect to the defaulting pages only via 4G, and not via wifi, in case it is a problem related to the ISP, but to no avail. Please note that JavaScript is enabled in all browsers. 

Not sure if there is a relevant connection here, but it may still be useful for you to know that these 4 above-mentioned pubs all have a "references.bib file", whereas 4 other pubs which don't have a "references.bib" file don't encounter any rendering issues on the same Mac devices. Also, the MacMini where all the pubs do render have BasicTex installed on it. 

Thank you very much in advance for your help.

Kimmichka.

[PubPub01.pdf](https://github.com/pubpub/pubpub/files/1151288/PubPub01.pdf)
[PubPub02.pdf](https://github.com/pubpub/pubpub/files/1151289/PubPub02.pdf)
[PubPub03.pdf](https://github.com/pubpub/pubpub/files/1151290/PubPub03.pdf)
[PubPub04.pdf](https://github.com/pubpub/pubpub/files/1151291/PubPub04.pdf)

**Comment by Kimmichka** [comment]
Hello!
You may be interested to know that updating the iPhone SE from iOS 10.3.2 to iOS 10.3.3 allows the pubs to render for the first time on this device.
Thank you.
Kimmichka.

**Comment by ThariqS** [comment]
Thank you Kimmichka! Does this rendering problem still exist on the other devices you mentioned?

Being a fairly small development team we often don't have the resources to comprehensively test the code on a variety of devices so this is useful. It does seem like the pattern is for pubs with references. We're currently using a library to handle that on the fly which may not be supported in older browsers, but we're moving to doing that functionality on the server so it should be more reliable across devices.

**Comment by Kimmichka** [comment]
Hi Thariq!
I'm glad that my bug report can prove useful. 
Yes, I tried again yesterday on all the other mentioned devices and the same rendering problem still persists. 
But I updated the iPad Mini 3 this morning from iOS 9.3.5 to iOS 10.3.3 and, just as it happened with the iPhone SE, all the pubs with references render now for the first time, so it does seem like indeed that the Mac & iOS older browsers cannot show pubs with references. 
Hopefully, moving the references functionality to the server will be of help. :)
Thank you very much.
All the best,
Kimmichka.
